inc/sccmxml.class.php

```diff
@@ -323,6 +323,32 @@ function setUsers() {
       $USERS->addChild('LOGIN', $this->username);
    }
 
+   function determineNetworkType($networkDescription) {      
+      $description = strtolower($network_description);
+
+      $networkTypes = [
+        'wifi' => ['wi-fi', 'wireless', 'wifi'],
+        'infiniband' => ['infiniband'],
+        'aggregate' => ['aggregation', 'aggregate'],
+        'alias' => ['alias'],
+        'dialup' => ['dialup', 'dial-up'],
+        'loopback' => ['loop'],
+        'bridge' => ['bridge'],
+        'fibrechannel' => ['fibre', 'fiber'],
+        'bluetooth' => ['bluetooth'],
```

Comment:
```suggestion
        'wifi' => ['wi-fi', 'wireless', 'wifi'],
        false => ['infiniband'], //no infiniband specialisation yet on GLPI
        'aggregate' => ['aggregation', 'aggregate'],
        'alias' => ['alias'],
        'dialup' => ['dialup', 'dial-up'],
        'loopback' => ['loop'],
        false => ['bridge'], //no bridge specialisation yet on GLPI
        'fibrechannel' => ['fibre', 'fiber'],
        false => ['bluetooth'], //no bluetooth specialisation yet on GLPI
```

Reply:
I suggest not setting a XML node ```<TYPE>``` for ```bluetooth```, ```infiniband``` and ```bridge```

As there is no dedicated specialisation on GLPI

Can you retry with this change  and see what it looks like on GLPI for the bluetooth device